src/Tests/Eshopworld.Messaging.Tests/MessengerTopicTest.cs

```diff
@@ -83,6 +83,29 @@ public async Task Test_ReceiveCreatesTheTopic()
         }
     }
 
+    [EswEventName("test.prova.nino")]
+    class TestNinoEvent : DomainEvent
+    {
+
+    }
+
+    [Theory, IsLayer1]
+    [InlineData(typeof(TestEventWithoutAttribute))]
+    [InlineData(typeof(TestEventWithAttribute))]
+    [InlineData(typeof(TestEventWithEmptyAttributeValue))]
+    [InlineData(typeof(TestEvenWithWhiteSpaceAttributeValue))]
+    public async Task Test_SendingNamedEvents(Type type)
+    {
+        await ServiceBusFixture.ServiceBusNamespace.ScorchNamespace();
+
+        using (IDoFullMessaging msn = new Messenger(ServiceBusFixture.ConfigSettings.ServiceBusConnectionString, ServiceBusFixture.ConfigSettings.AzureSubscriptionId))
+        {
+            dynamic message = Activator.CreateInstance(type);
+            await msn.Publish(message);
+            ServiceBusFixture.ServiceBusNamespace.AssertSingleTopicExists(type);

```

Comment:
in principle this test should use explicit name cause if you had a bug in get entity name call you could have your test succeeding but invalid topic created...